presto-kudu/src/main/java/com/facebook/presto/kudu/properties/KuduTableProperties.java

```diff
@@ -66,6 +66,7 @@
     public static final String PARTITION_BY_RANGE_COLUMNS = "partition_by_range_columns";
```

Comment:
can you add a test where you actually access these properties (maybe some SHOW CREATE TABLE)?

Reply:
Okay - done.